ftw/solr/browser/configure.zcml

```diff
@@ -1,8 +1,15 @@
 <configure
     xmlns="http://namespaces.zope.org/zope"
+    xmlns:zcml="http://namespaces.zope.org/zcml"
     xmlns:browser="http://namespaces.zope.org/browser"
+    xmlns:theme="http://namespaces.zope.org/plonetheme.onegov"
```

Comment:
The namespace definition does not fail even when `plonetheme.onegov` isn't installed.
I tested that locally. Also the tests would fail at startup if there was some invalid zcml.